### ISSUE TITLE
[Bug Report] VAvatar inside small VChip is not sized properly

### ISSUE THREAD

**Issue by Dylan-Chapman** [vuetifyjs/vuetify#8129]
### Environment
**Vuetify Version:** 2.0.2
**Vue Version:** 2.6.10
**Browsers:** Chrome 75.0.3770.142
**OS:** Windows 10

### Steps to reproduce
Create a `VChip` component with the `small` prop and a child `VAvatar` 

### Expected Behavior
`VAvatar` has spacing between itself and the edge of the `VChip` just like it does when the `small` prop isn't set

### Actual Behavior
`VAvatar` goes to the edge of the `VChip`

### Reproduction Link
<a href="https://codepen.io/Dylan-Chapman/pen/QedyQg" target="_blank">https://codepen.io/Dylan-Chapman/pen/QedyQg</a>

### Other comments
This also can't be fixed by setting the `VAvatar`'s `size` prop because of `!important` style declarations.

**Comment by Dylan-Chapman** [comment]
Don't know if it's worth making a separate issue over, but `VAvatar` is also over-sized when the `VChip` has both `small` and `pill` props set.

**Comment by jacekkarczmarczyk** [comment]
You can add this other issue to the codepen

**Comment by Dylan-Chapman** [comment]
The codepen has been updated to show both of these issues.

**Comment by sh7dm** [comment]
The problem is present with any non-default sizes:
![image](https://user-images.githubusercontent.com/19504461/62390348-b4715d80-b56a-11e9-9438-dc1996cbe232.png)
Codepen: https://codepen.io/sh7dm/pen/KOXdBe
Playground:
<details>

```vue
<template>
  <v-container>
    <v-layout justify-space-around wrap align-space-around>
      <v-flex>
        <v-chip x-small>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
        <v-chip x-small pill>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
      </v-flex>

      <v-flex>
        <v-chip small>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
        <v-chip small pill>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
      </v-flex>

      <v-flex>
        <v-chip>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
        <v-chip pill>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
      </v-flex>

      <v-flex>
        <v-chip large>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
        <v-chip large pill>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
      </v-flex>

      <v-flex>
        <v-chip x-large>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
        <v-chip x-large pill>
          <v-avatar color="blue">A</v-avatar>
          Text
        </v-chip>
      </v-flex>
    </v-layout>
  </v-container>
</template>

<script>
export default {
  data: () => ({
  })
}
</script>
```
</details>

**Comment by sh7dm** [comment]
also I noticed that avatar is positioned incorrectly

**Comment by bykka** [comment]
@sh7dm for some cases helps property "left" for v-avatar.
`<v-avatar left>...`

**Comment by johnleider** [comment]
resolved by #13192

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
